src/lib/components/details/ProductDetails.svelte

```diff
@@ -0,0 +1,101 @@
+<script lang="ts">
+	import { cart } from '$lib/stores/cart';
+	import { Button } from '$lib/components/ui/button';
+	import type { Product } from '../../../utils/products';
+
+	export let product: Product;
+	export let selectedSize = '';
+
+	product.name = product.name.split('#')[0].trim();
+
+	function addItemToCart() {
+		cart.addItem(product, selectedSize);
+	}
+</script>
+
+<div
+	class="mx-auto mb-16 mt-[54px] flex w-full flex-col justify-center gap-8 px-4 sm:px-8 md:mt-[80px] md:flex-row md:px-16"
+>
+	<!-- Product Image -->
+	<div
+		class="card aspect-[4/3] w-full max-w-[765px] overflow-hidden rounded-2xl border border-gray-200 bg-white"
+	>
+		<img
+			alt={product.name}
+			src={product.imageUrls[0]}
+			class="h-full w-full rounded-lg object-cover"
+		/>
+
+		<!-- TODO: Add carousel buttons		-->
+	</div>
+
+	<!-- Product Details -->
+	<div class="flex w-full max-w-lg flex-col gap-6 md:max-w-[379px]">
+		<!-- Product Name -->
+		<h1
+			class="font-inter text-[48px] font-normal leading-[68px] tracking-[-0.64px] text-[#19191C] md:text-[64px]"
+		>
+			{product.name}
+		</h1>
+
+		<!-- Product Price -->
+		<span
+			class="font-inter text-[16px] font-normal leading-[140%] tracking-[-0.16px] text-[#56565C]"
+		>
+			{product.price.currency}{product.price.amount.toFixed(2)}
+		</span>
+
+		<hr class="bg-[#EDEDF0]" />
+
+		<!-- Product Description -->
+		<p class="font-inter text-[16px] font-normal leading-[22px] text-[#56565C]">
+			{product.description}
```

Comment:
Can we support markdown here?

Reply:
Example?

Reply:
One could do
```
description: `
# BEST PRODUCT EVER!

Our product supports:
- 24/7 uptime
- left-handed holding
- up to 5 faster speed than previous model

Used by [Google](google.com) and [Facebook](meta.com).
```

It gives possibilities to do more using configs witohut having to go into code.

I think Tailwind has a class for it, or a plugin. Purse, from what I recall

Reply:
aahh, I think I could do.